### PR TITLE
add node env

### PR DESCRIPTION
**Pull request by sophia0328**
Signed-off-by: Gong Sophia <sophia.gong@intel.com>

### CODE REVIEW

File: node/Dockerfile [hunk 1]
```diff
@@ -33,6 +33,10 @@ FROM clearlinux/os-core:latest
 
 COPY --from=builder /install_root /
 
+ENV NAME="node" \
+    VERSION="v12.13.0" \
```
Comment (qzheng527) [inline node/Dockerfile]: In Clear, the version is changing from time to time. How can you make sure it is always v12.13.0?
And if no automatic way, how can you make sure your manual update is always in time?

File: node/Dockerfile [hunk 2]
```diff
@@ -33,6 +33,10 @@ FROM clearlinux/os-core:latest
...
+    PATH="/usr/bin:$PATH"
```
Comment (qzheng527) [inline node/Dockerfile]: The clear already had the env PATH defined. What is the value here to add one new PATH?

Comment (sophia0328) [reply node/Dockerfile]: will skip this info as it's already provided.

**Comment by sophia0328** [comment]
please review https://github.com/clearlinux/dockerfiles/pull/301

**Pull request closed without merging**